pkg/printers/internalversion/printers_test.go

```diff
@@ -6206,6 +6208,322 @@ func TestPrintFlowSchema(t *testing.T) {
 	}
 }
 
+func TestPrintDeviceClass(t *testing.T) {
+
+	tests := []struct {
+		deviceClass resourceapis.DeviceClass
+		expected    []metav1.TableRow
+	}{
+		{
+			deviceClass: resourceapis.DeviceClass{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "deviceclass",
+					CreationTimestamp: metav1.Time{Time: time.Now().Add(1.9e9)},
```

Comment:
How do we guarantee that `time.Now().Add(1.9e9)` will always be translated to `0s` to prevent any flakes?.